src/pages/Detail/components/BuyButton.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import { flexBox } from '../../../styles/mixin';
+
+const BuyButton = () => {
+  return (
+    <BuyBtn>
+      <BtnTitle>구매</BtnTitle>
+      <BtnText>
+        <BtnPrice>123,123원</BtnPrice>
+        <BtnType>즉시 구매가</BtnType>
+      </BtnText>
+    </BuyBtn>
+  );
+};
+
+export default BuyButton;
+
+const BuyBtn = styled.a`
+  ${flexBox('', 'center', '')}
```

Comment:
저희 flexBox default값으로 center를 주는걸로 구현되어있으니 이렇게 수정해도 될 것 같습니다. 
```suggestion
  ${flexBox()}
```